### PR TITLE
Feature/process file

### PR DESCRIPTION
**Pull request by st3v3nhunt**
This is good for review, additional info has been added to the `README` about running the app.

More tests are needed which I'll be working on next along with looking at how to get this data into the db.

I'm also interested in seeing whether the file can be streamed rather than downloading initially.

### CODE REVIEW

File: app/lib/downloadFile.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const log = require('./logger');
+const apiRequest = require('./apiRequest');
+const fsHelper = require('./fsHelper');
+const constants = require('./constants');
+
+const POMI_FILE = constants.POMI_FILE;
+const POMI_DOWNLOAD_LOCATION = constants.POMI_DOWNLOAD_LOCATION;
+
+function downloadFile() {
+  return new Promise((resolve, reject) => {
```
Comment (markysoft) [inline app/lib/downloadFile.js]: The API request already returns a promise, the wrapping promise isn't required and the body could be written as 

```
 log.time('Downloading the POMI file took');
 return apiRequest(POMI_DOWNLOAD_LOCATION)
   .then((data) => {
     log.timeEnd('Downloading the POMI file took');
     fsHelper.saveFileSync(data, POMI_FILE);
   });
```

File: app/lib/getLatestPeriod.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+      resolve(latestPeriod);
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+module.exports = getLatestPeriod;
+
```
Comment (markysoft) [inline app/lib/getLatestPeriod.js]: As this method is called from a previous promise it is auto-magically wrapped in a promise so a new Promise isn't required. The body could just be
```
  log.time('Getting latest period took');
  const latestPeriod = utils.getMostRecentPeriod(periods);
  log.timeEnd('Getting latest period took');
  log.info(`Latest period is ${latestPeriod}`);
  return latestPeriod;
```

**Review by markysoft — CHANGES_REQUESTED** [review]
Suggestions to simplify promise handling

**Comment by st3v3nhunt** [comment]
@markysoft Changes made.

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**